jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java

```diff
@@ -74,16 +75,16 @@ public String getBaggageItem(String key) {
     return this.baggage;
   }
 
-  public long getTraceID() {
-    return traceID;
+  public long getTraceId() {
```

Comment:
Can we keep the methods with uppercase ID as deprecated and disable stylecheck on them? We have internal libraries that I don't want to break by a non-essential renaming.